### PR TITLE
podman-remote pull

### PR DESCRIPTION
**Pull request by baude**
Add status for remote users and podman remote-client pull.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM, but needs a rebase.

File: pkg/varlinkapi/images.go [hunk 1]
```diff
@@ -609,24 +604,74 @@ func (i *LibpodAPI) PullImage(call iopodman.VarlinkCall, name string, certDir, c
...
+			}
+			newImage, err := i.Runtime.ImageRuntime().LoadFromArchiveReference(getContext(), srcRef, signaturePolicy, output)
+			if err != nil {
+				c <- errors.Wrapf(err, "error pulling image from %q", name)
+			}
+			imageID = newImage[0].ID()
+		} else {
+			newImage, err := i.Runtime.ImageRuntime().New(getContext(), name, signaturePolicy, "", output, &dockerRegistryOptions, so, false, nil)
+			if err != nil {
+				c <- errors.Wrapf(err, "unable to pull %s: %s", name)
```
Comment (TomSweeneyRedHat) [inline pkg/varlinkapi/images.go]: missing a variable here, you've two '%s'

**Comment by TomSweeneyRedHat** [comment]
One nit that's probably mucking up the tests, otherwise LGTM

**Comment by baude** [comment]
ready for merge

**Comment by TomSweeneyRedHat** [comment]
@baude looks like you need to rebase this one.

**Pull request merged**